modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-free/en-US.html

```diff
@@ -129,8 +129,8 @@ <h2 style=" font-size: 30px; font-weight: bolder; margin-top: 2.5rem; margin-bot
 						</div>
 
 						<div style=" padding: 10px; text-align: center; margin: 2.5rem; ">
-							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem; ">Go to Dashboard</a>
-							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold; ">Go to Cloud Console</a>
+							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem; " hidden>Go to Dashboard</a>
+							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold; ">Continue Configuration</a>
```

Comment:
Please, remove the spaces in this line.
```suggestion
							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold;">Continue Configuration</a>
```